backend/clubs/views.py

```diff
@@ -2611,6 +2612,8 @@ def create_tickets(self, request, *args, **kwargs):
                                         group_discount:
                                             type: number
                                             required: false
+                                        transferable:
```

Comment:
Are you using this anywhere in `create_tickets`?

Reply:
Forgot to add back after the merge, thanks for the catch